source/css/_objects.locator.scss

```diff
@@ -0,0 +1,385 @@
+////
+/// Church Locator Components
+////
+
+/// Root Map Container
+///
+/// @atoms/map/google-map
+/// @pages/locator/*
+#map {
+  width: 100%;
+  flex: 1;
+  height: rem(670);
+}
+
+.c-map-main {
+  display: flex;
+  flex-direction: column;
+
+  .c-map-container {
+    margin: 0;
+    width: 100%;
+    flex: 1;
+    display: flex;
+    height: rem(670);
+  }
+}
+.c-map-main--2col {
+  display: grid;
+  grid-template-columns: col-width(6);
+  grid-template-rows: fit-content(#{rem(20)}) auto;
+  grid-template-areas:
+    "header"
+    "map"
+    "side";
+
+  @include media(">large") {
+    grid-template-columns: col-width(2) col-width(4);
+    grid-template-areas:
+    "header header"
+    "side map";
+  }
+  @include media(">xlarge") {
+    grid-template-columns: col-width(1) col-width(2) col-width(3);
+    grid-template-areas:
+    "header header header"
+    "gap side map";
+  }
+
+  .c-page-header {
+    grid-area: header;
+  }
+  .c-search-results {
+    grid-area: side;
+
+    @include media(">large") {
+      overflow-y: scroll;
+      height: rem(670);
+    }
+  }
+  .c-map-container {
+    grid-area: map;
+    margin: 0;
+    width: 100%;
+    flex: 1;
+    display: flex;
+    height: rem(670);
+  }
+}
+
+/// Church Info Popup
+///
+/// @molecules/church-locator/church-info
+/// @molecules/church-locator/church-info-large
+.c-church-info.c-church-info--basic {
+  max-width: rem(400);
+  width: 100%;
+  max-height: rem(700);
+  background: $c-white;
+
+  .c-church-info__wrap {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+    padding: rem(10) rem(40) rem(10) rem(20);
+  }
+
+  .c-church-info__title {
+    font-family: $ff-font--primary;
+    font-weight: bold;
+    font-size: rem(20);
+    line-height: rem(24);
+  }
+
+  .c-church-info__location {
+    font-family: $ff-font;
+    font-size: rem(14);
+    line-height: rem(19);
```

Comment:
The font styling here needs to be replaced with use of the already existing font class `u-font--secondary--s`